WordPress/Sniffs/VIP/DirectDatabaseQuerySniff.php

```diff
@@ -167,7 +167,8 @@ public function process( PHP_CodeSniffer_File $phpcsFile, $stackPtr ) {
 		}
 
 		$whitelisted_cache = false;
-		$cached            = $wp_cache_get = false;
+		$cached            = false;
+		$wp_cache_get      = false;
```

Comment:
Oh, there's a quite simple explanation for that:
In `WordPress/Sniff.php` we're assigning the output of a _function call_ to a variable and if a second variable needs the same value, there is no need to repeat the function call (efficiency).
In the above code, we're assigning the PHP native constant `false`, which carries no overhead in contrast to the function call.